scripts/render_configs.sh

```diff
@@ -68,12 +72,18 @@ if $USE_VAULT; then
 else
   AZURE_SECRETS=$(gcloud secrets versions access latest --project $GCLOUD_PROJECT --secret azure-secrets)
 fi
-
+# These are used in the variable echoing or setting to clipboard below
+# shellcheck disable=SC2034
 AZURE_CREDENTIALS_HOMETENANTID=$(echo "$AZURE_SECRETS" | jq -r '."tenant-id"' | tee /tmp/jade-dev-tenant-id.key)
```

Comment:
Is there still something using the tee'd output files? I don't see it used anymore.

Reply:
I don't know, but I am hesitant to change render-configs now as part of this PR, beyond what is needed to bring in the scripts.